assets/chezmoi.io/docs/reference/commands/add.md

```diff
@@ -102,6 +102,6 @@ $ chezmoi add ~/.oh-my-zsh --exact --recursive
 
     `chezmoi add` will fail if the entry being added is in a directory
     implicitly created by an
-    [external](../special-files-and-directories/chezmoiexternal-format.md).
+    [external](../special-files/chezmoiexternal-format.md).
```

Comment:
I wonder if there is a way to verify that all links are valid automatically.

Reply:
I think I didn't pay enough attention to the `serve-docs` console. Probably worth enabling [`strict`](https://www.mkdocs.org/user-guide/configuration/#strict) in mkdocs.

Reply:
Great spot. Added in 69ced411df2f173a6cbf4815273e10b3268d442c.